commit ce6e043a44e35144b038bc0236b5d244ffee1ca5
Author: Wilson E. Alvarez <wilson.e.alvarez1@gmail.com>
Date:   Tue Mar 27 22:45:14 2018 -0400

    src: Added const references to various function parameters
    
    Added const references to various function parameters in order to avoid
    copying data unnecessarily and enhancing performance
    
    Signed-off-by: Wilson E. Alvarez <wilson.e.alvarez1@gmail.com>

diff --git a/src/mgr/PyModuleRunner.h b/src/mgr/PyModuleRunner.h
index 40b27d11f20..b4362fae162 100644
--- a/src/mgr/PyModuleRunner.h
+++ b/src/mgr/PyModuleRunner.h
@@ -53,7 +53,7 @@ public:
   void log(int level, const std::string &record);
 
   PyModuleRunner(
-      PyModuleRef py_module_,
+      const PyModuleRef &py_module_,
       LogChannelRef clog_)
     : 
       py_module(py_module_),
